### ISSUE TITLE
implement `new_child` method in Span class

### ISSUE THREAD

**Issue by konstantin-stepanov** [aio-libs/aiozipkin#83]
I think that would be useful. Because now I have to write something like:

```python
with parent_span.tracer.new_child(parent_span.context) as span:
    span.name('some:action')
    span.kind(CLIENT)
    # do something
```

I propose to define a method `new_child` in Span class(also in SpanAbs and SpanNoop) like:
```python
class Span(SpanAbc):

    def new_child(self, name: str = None, kind: str = None) -> 'Span':
        span = self.tracer.new_child(self.context)
        if name is not None:
            span.name(name)
        if kind is not None:
            span.kind(kind)
        return span

    ...
```
then it could be used like:
```python
with parent_span.new_child('some:action', CLIENT) as span:
    # do some thing
```

What do you think about it?

**Comment by jettify** [comment]
I like the idea, would you like to implement this feature and tests?

**Comment by konstantin-stepanov** [comment]
Yes, l`ll implement it

**Comment by jettify** [comment]
Fixed in #84 new version released https://pypi.python.org/pypi/aiozipkin/0.1.1
